grammars/julia.cson

```diff
@@ -246,11 +246,15 @@ repository:
         name: "keyword.operator.range.julia"
       }
       {
+        match: "(?:\\|>)"
+        name: "keyword.operator.applies.julia"
+      }
+      {
         match: "(?:\\||\\&|~)"
         name: "keyword.operator.bitwise.julia"
       }
       {
-        match: "(?:\\+|-|\\*|\\.\\*|/|\\./|//|\\.//|%|\\.%|\\\\|\\.\\\\|\\^|\\.\\^)"
+        match: "(?:\\+\\+|--|\\+|\\.\\+|-|\\.\\-|\\*|\\.\\*|//(?!=)|\\.//(?!=)|/|\\./|%|\\.%|\\\\|\\.\\\\|\\^|\\.\\^)"
```

Comment:
Again, are these lookaheads necessary?  The ones in `keyword.operator.update` should definitely be getting precedence over these.